### ISSUE TITLE
Orphaned blocks

### ISSUE THREAD

**Issue by alexsorkin** [israelcoin/Isracoin#12]
In last 2 days we experienced known issue from Doge.
See following bug reports:

https://github.com/dogecoin/dogecoin/issues/232
https://github.com/dogecoin/dogecoin/issues/208

We are getting about 1 Orphaned block per 1000.
The cause seems to be:
USE_UPNP=1

**Comment by alexsorkin** [comment]
Example debug.log entry for block 14252:

2014-04-01 20:20:24 ThreadRPCServer method=getmininginfo
2014-04-01 20:20:24 ThreadRPCServer method=getdifficulty
2014-04-01 20:20:24 ThreadRPCServer method=getblockcount
2014-04-01 20:20:25 received block df590071e9ad86565cf17822d661327823edfd5901f300231f4ae0cfd74e0243
2014-04-01 20:20:25 ProcessBlock: ORPHAN BLOCK, prev=5ddd18f665f11e9dc265aa4e3154698a284fbb2e990b7dd1ce83cb46c8e3f439
2014-04-01 20:20:26 received block 5ddd18f665f11e9dc265aa4e3154698a284fbb2e990b7dd1ce83cb46c8e3f439
2014-04-01 20:20:26   nActualTimespan = 212  before bounds

**Comment by israelcoin** [comment]
Hmm. Is this due to the Pool's Isracoind having UPNP on or because of wallets in the network?

**Comment by alexsorkin** [comment]
No, no UI wallet in network.
isracoind

I'm hosting the server on Amazon...so most probably (for sure) UPNP is not available.

see also http://p2pool-il.tk:1948/static/
P2Pool has same issue.

**Comment by alexsorkin** [comment]
I've tried to disable it via isracoin.conf using upnp=0.
But it doesn't work either (at all), blocks confirmations are not coming.

**Comment by guysoft** [comment]
Strange. 
On http://isr.nonstopmine.com/ the radio is on 1 in 2369 .
If I compare it to https://dogehouse.org its less by two orders of magnitude.

**Comment by israelcoin** [comment]
you can compile the isracoind this way:
cd ./src
make -f makefile.unix USE_UPNP=0 USE_IPV6=1
And it will compile without upnp at all.

**Comment by alexsorkin** [comment]
I've tried to compile it prior to official launch.
The source was missing genesis block.
BTW this was a major reason for late pools launch.

Was genesis block now added?

**Comment by guysoft** [comment]
Yes, you can see the commit message: https://github.com/israelcoin/Isracoin/commit/ca79e0700958e4c3e5a59019725a3a08b72f0d1b

**Comment by alexsorkin** [comment]
Ok...I'll try in the evening...ISR is not yet paying me salary.

**Comment by guysoft** [comment]
... Same here :-/
volunteer

Please update when you do

**Comment by alexsorkin** [comment]
I think Doge has more Orphanes simply due to higher network hashrate, in same order of magnitude.
There is linear dependency with entire coin network latency.
UPNP=0 should help much with latency, according to the Doge reports comments.
